platforms/web/yarn.lock

```diff
@@ -4450,14 +4616,26 @@ vite-plugin-dts@^1.5.0:
   optionalDependencies:
     fsevents "~2.3.2"
 
+"vite@^3.0.0 || ^4.0.0":
+  version "4.1.4"
+  resolved "https://registry.yarnpkg.com/vite/-/vite-4.1.4.tgz#170d93bcff97e0ebc09764c053eebe130bfe6ca0"
+  integrity sha512-3knk/HsbSTKEin43zHu7jTwYWv81f8kgAL99G5NWBcA1LKvtvcVAC4JjBH1arBunO9kQka+1oGbrMKOjk4ZrBg==
+  dependencies:
+    esbuild "^0.16.14"
+    postcss "^8.4.21"
+    resolve "^1.22.1"
+    rollup "^3.10.0"
+  optionalDependencies:
+    fsevents "~2.3.2"
+
 vitest-sonar-reporter@^0.3.3:
   version "0.3.3"
   resolved "https://registry.yarnpkg.com/vitest-sonar-reporter/-/vitest-sonar-reporter-0.3.3.tgz#10e3761144be2cd8efb4fee88265ff919de67c1d"
   integrity sha512-OvH+AYMgDFk3SK7sohn5KUHUOhB42tWgSTW7YzB/v/dN3x+7bgXD2L0FZpDYjC1hvBMJJ4HFqOsiO9LsETmgxg==
   dependencies:
     pathe "^0.3.9"
 
-vitest@0.23.4, vitest@^0.23.4:
+vitest@0.23.4:
```

Comment:
The old version shouldn't be necessary with the upgrade

Reply:
To sort this out I think it's necessary to upgrade vite and the react plugin, so I've done that in the most recent commits.